@kiva/kv-components/src/vue/KvSideSheet.vue

```diff
@@ -144,23 +164,21 @@ export default {
 		},
 		/**
 		 * Source element position for expand animation
-		 */
+		 * */
```

Comment:
I don't know why/when we started adding these extra asterisks to the jsdocs in this repo... Haha. No change needed, just something funny.